.gitignore

```diff
@@ -121,3 +121,4 @@ docs/getting_started/index.md
 docs/getting_started/step_*/
 docs/getting_started/src/
 docs/getting_started/summary.txt
+docs/manuals/rest/*.json
```

Comment:
Probably the same rule for docs/manuals/rest/*.md

Reply:
Ok but exclude the index.md

Reply:
Right. Using the ! syntax, just like we already have already:
`docs/manuals/gui/viselements/*.md` except `index`.